packages/compiler-cli/ngcc/main-ngcc.ts

```diff
@@ -113,6 +120,9 @@ if (require.main === module) {
   const enableI18nLegacyMessageIdFormat = options['legacy-message-ids'];
   const invalidateEntryPointManifest = options['invalidate-entry-point-manifest'];
   const errorOnFailedEntryPoint = options['error-on-failed-entry-point'];
+  // yargs is not so great at mixed string+boolean types, so we have to test tsconfig against a
+  // string "false" to capture the `tsconfig=false` option
+  const tsConfigPath = (options['tsconfig'] === 'false') ? null : options['tsconfig'];
```

Comment:
Unless I am missing something, we also need to set `tsConfigPath` to `null` when `tsconfig` is negated with `--no-tsconfig`. So, I would expect this to be:

```suggestion
  const tsConfigPath = (options['tsconfig'] === false || options['tsconfig'] === 'false') ? null : options['tsconfig'];
```

...or something like...

```suggestion
  const tsConfigPath = (`${options['tsconfig']}` === 'false') ? null : options['tsconfig'];
```

Reply:
Darn it! You are right. Thanks.

Reply:
Note that in my test project I was doing:

```
console.log('force no tsconfig', '' + options.tsconfig === 'false')
```